bugmon/evaluator_configs/browser.py

```diff
@@ -0,0 +1,65 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+from pathlib import Path
+from typing import Union, Generator
+
+from autobisect import BrowserEvaluator
+
+from .base import BaseEvaluatorConfig
+from ..bug import EnhancedBug
+
+ALLOWED = ["*.htm", "*.html", "*.svg", "*.xml", "*"]
+EXCLUDED = ["*.js", "*.txt"]
+
+
+def identify_prefs(attachment_dir: Path) -> Union[Path, None]:
+    """Determine if the bug includes a prefs.js file
+
+    :param attachment_dir: Path to the downloaded attachments
+    :return:
+    """
+    prefs_path = None
+    for file_name in attachment_dir.iterdir():
+        if file_name.suffix == ".js":
+            file_path = Path(attachment_dir / file_name)
+            if "user_pref" in file_path.read_text():
+                prefs_path = file_path
+
+    return prefs_path
+
+
+class SimpleBrowserConfig(BaseEvaluatorConfig, BrowserEvaluator):
+    """Simple Browser Evaluator Configuration"""
+
+    @classmethod
+    def iterate(
+        cls, bug: EnhancedBug, working_dir: Path
+    ) -> Generator["SimpleBrowserConfig", None, None]:
```

Comment:
You could just do `Interator["SimpleBrowserConfig"]` ([typing docs](https://docs.python.org/3/library/typing.html#typing.Generator) under "Alternatively...")

Ditto in `SimpleJSConfig` and `BaseEvaluatorConfig`.